Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/NodeTraversal/CountNodes.feature

```diff
@@ -70,10 +70,11 @@ Feature: Find nodes using the countNodes query
       | b               | b        | Neos.ContentRepository.Testing:Page        | home                   | {"text": "b"}         | {}                                       |
       | b1              | b1       | Neos.ContentRepository.Testing:Page        | b                      | {"text": "b1"}        | {}                                       |
 
-  Scenario: count all nodes
+  Scenario:
+    # count all nodes
     When I execute the countNodes query I expect the result to be 12
 
-  Scenario: count all nodes with disabled nodes
+    # count all nodes with disabled nodes
```

Comment:
I just combined some scenarios with the same setup to speed up the tests a little